routers/repo/branch.go

```diff
@@ -203,10 +204,16 @@ func loadBranches(ctx *context.Context) []*Branch {
 			}
 		}
 
+		isMerged := false
+		if divergence.Ahead == 0 && divergence.Behind > 0 && ctx.Repo.Repository.DefaultBranch != branchName {
```

Comment:
And I thought it again, maybe `divergence.Behind > 0` should be `divergence.Behind >= 0`? Consider a branch just merged to `master` and no new commits pushed to master.

Reply:
but this also let branches who are clones of default-branch show up as "merged":

```
git checkout -b testbranch origin/master
git push -u origin testbranch
```